content/03.aim1.md

```diff
@@ -29,4 +29,37 @@ We will evaluate resulting models for their applicability to rheumatic disease a
 
 ### Results
 
-TBD.
+#### VAE test on simulated single cell datasets
+#### 1. Simulation data generation: 
+Simulated single cell data was generated by splatter [@doi:10.1186/s13059-017-1305-0]. 
+
+Parameters used:
+* nCells - The number of cells to simulate: 500 - 5000
+* nGenes - The number of genes to simulate: 20000 - 60000
+* nGroups - The number of celltypes: 5 - 15
+* outlier - probability of a gene that is an expression outlier:  0.1 - 0.5
+* default parameter is ncells = 600, nGenes = 20000, celltypes = 5, batchsize = 1
+
+Simulation statistics under different parameters:
+
+![](images/mean.gene.exp.bmp)
+
+Figure 1: Distribution of mean expression level of simulated genes across samples under different simulated parameters.
+
+#### 2. Visualization of simulated single cell data using VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE and PCA under different parameters
+
+To see if different methods can recover cell types, we compared the 2D visualizations on simulated datasets. <br />
```

Comment:
I am confused why the `<br />` are added here? I think @cgreene 's [comment](https://github.com/greenelab/czi-hca-report/pull/1#discussion_r173277856) meant to make each sentence a new line in the markdown file, not necessarily appear as a new line in the rendered version :man_shrugging:

Reply:
Yep - just new line in the markdown file. Not adding explicit line breaks 😄